### ISSUE TITLE
[Suggestion] Something for likes on a tweet?

### ISSUE THREAD

**Issue by Dream-Lair** [satsukitv/twitter-block-chain#12]
Eg., a racist tweets something racist, and hundreds of people like the tweet. Those people are probably also racist, but might not necessarily be following the original tweeter. It'd be nice to block all of them at once.

**Comment by twilight-sparkle-irl** [comment]
Agreed, same for retweets. I was just wondering if this was possible as well. It'd be very useful for things like 4chan troll campaigns where the account, once again, might not necessarily be following instigators to keep a low profile, but will signal boost the troll tweets.

**Comment by ceceradio** [comment]
its worth investigating but i can't give an ETA for this feature. it'd require some significant changes to the code

**Comment by twilight-sparkle-irl** [comment]
I've done some investigating -- it appears that it only shows you a limited amount of likes/retweets each time you open the dialog, and you'd have to reopen and reclose it each and every time to get all the likes and retweets. With [twhack](https://abs.tw1tter.com/), you can simply disable the block dialog popup and run `$('.activity-popup-dialog-users .block-text.not-blocked').click()` in the console, then manually reopen it each time until you've cleared all the users out. I wouldn't know how to go about bypassing the block dialog quickly from a userscript standpoint as opposed to an injector standpoint.
